pymeasure/instruments/hp/__init__.py

```diff
@@ -24,3 +24,5 @@
 
 from .hp33120A import HP33120A
 from .hp34401A import HP34401A
+from .hp3478A import HP3478A
+
```

Comment:
This file is not part of your this pull request and it is actually part of the master branch.
Again, something wrong with rebase ?

Reply:
messed up the rebase